tetris.py

```diff
@@ -127,10 +127,27 @@ def new_block(self):
             "I": [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
         }
         if self.upcoming_block is None:
```

Comment:
Change `upcoming_block` to `upcoming_block_shape`, because only the shape of the next block is meaningful, not coordinates:

```suggestion
        if self.upcoming_block_shape is None:
```